pkg/operators/operators.go

```diff
@@ -19,19 +19,41 @@ import (
 	"fmt"
 	"sync"
 
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	log "github.com/sirupsen/logrus"
 
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/datasource"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-service/api"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/logger"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/params"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
+type Type int
+
+const (
+	TypeImage = 1 << iota
+	TypeData
+	TypeTarget
```

Comment:
I am not sure where it is used, but why is it a bitfield?

Also it seems defined with the wrong type (`int` instead of `Type`)

Reply:
Agree in regards to `Type` - BitField because an operator could do multiple things. I think this can be removed now that I added more distinct interfaces for the different operator types.